0-good-advice/notes/what-i-wish-someone-had-told-me.md

```diff
@@ -0,0 +1,72 @@
+Read it yourself: https://blog.samaltman.com/what-i-wish-someone-had-told-me
```

Comment:
What if anything should be automated here? Article ingestion and formatting.

What tool would help us process links that we could add to with a variety of unstructured content?